### ISSUE TITLE
[Bug] IOS/Android/Desktop -  Messages are showing sent multiple times for sender

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#10550]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Launch App and login
2. Off WiFi
3. Send some messages to an account you have access too. (Preferably you're logged in on another device with)
4. On WiFi

## Expected Result:
Messages sent without internet connection should be sent without any issues.

## Actual Result:
Messages are sent twice for sender

## Workaround:
Uknown

## Platform:

Where is this issue occurring?

- iOS
- Android

**Version Number:** 1.1.89.0

**Reproducible in staging?:** Yes

**Reproducible in production?:** No

**Email or phone of affected tester (no customers):** any

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://user-images.githubusercontent.com/93399543/186508643-d20f6f4b-064f-479b-8dff-817ead6d2629.mp4

https://user-images.githubusercontent.com/93399543/186509768-320ad5cf-2bdb-4742-8880-a27bd5f38b40.MP4

**Expensify/Expensify Issue URL:** 

**Issue reported by:** Applause - Internal Team

**Slack conversation:** 

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by AndrewGable** [comment]
Few thoughts:
1. Agree I cannot reproduce on web
2. It seems(??) like a client side only issue since the double message is only seen on iOS and not on web
3. I think you have to leave and re-enter the page for the double message to show

**Comment by srikarparsi** [comment]
I agree, I wasn't able to reproduce this on web either. In the second video though, it looks like the message is rendering twice in the app without the user leaving the page.

**Comment by srikarparsi** [comment]
I'm un-assigning myself as I don't have much knowledge yet in ios/android and don't think I will be able to figure it out quick enough but will take a look back at the issue in the morning.

**Comment by AndrewGable** [comment]
OK we've decided to unblock this and move on. It requires a large-ish fix that is "moving backwards" in terms of where we want to go in terms of optimistic report action IDs/sequenceNumbers, so we are going to live with the bug for a little bit, then make sure it's fixed once we have removed sequenceNumbers.

**Comment by kavimuru** [comment]
@AndrewGable Able to reproduce in Android (1.1.97-2).  iOS build is still 96.5 and able to reproduce.

https://user-images.githubusercontent.com/43996225/188719348-8a3137e1-6b6d-40ed-9e05-f0ac58e620b2.mp4

**Comment by AndrewGable** [comment]
Ok thank you - I believe the fix is still in the works.

**Comment by mallenexpensify** [comment]
This appears to also be happening on desktop and it's semi-consistent. Thread, examples and logs here
https://expensify.slack.com/archives/C01GTK53T8Q/p1663717163964739?thread_ts=1663714013.290919&cid=C01GTK53T8Q

@JmillsExpensify I think you should add this to WA Project. I'll keep digging for more examples (inc. web) and reproducible steps

**Comment by JmillsExpensify** [comment]
Yes, agreed. I'll add it as this keeps coming up.

**Comment by mallenexpensify** [comment]
New bits from testing - [Slack post](https://expensify.slack.com/archives/C01GTK53T8Q/p1663872947476579?thread_ts=1663714013.290919&cid=C01GTK53T8Q)

Here's an interesting tidbit...

1. Noticed I had multiple of the same message show from me to Scott on Desktop Version 1.2.5-0
2. Scrolled up to see how many were showing and to get screenshot
3. Noticed as I scrolled (both ways?) that new duplicates were loading

Seems like it's related to loading or _maybe_ pagination? (guessing). Vid and logs.
[scroll to make duplicate message.txt](https://github.com/Expensify/App/files/9628666/scroll.to.make.duplicate.message.txt)


https://user-images.githubusercontent.com/22508304/191829517-f0a5069c-2188-4865-9fc5-927ecc715baf.mp4

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by trjExpensify** [comment]
I'm going to update the title of this issue to put `[Hold Sequence #]` in it as I understand we're held here on implementing that project.

**Comment by AndrewGable** [comment]
On hold melvin!

**Comment by AndrewGable** [comment]
Going to close as this will be fixed by sequence numbers. Reopen if you disagree.

**Comment by mallenexpensify** [comment]
I think we should leave all relevant issues open for larger projects so that, once the fix is implemented, we can test all variations to ensure the bug is fixed.  Is there a reason we wouldn't do this?  Since it's on hold, you can bump it to `Monthly` to minimize pings

**Comment by JmillsExpensify** [comment]
I'm kind of on the fence. I would almost rather track this issues elsewhere than keep them open in the repo. The focus here should be on bugs and performance improvements we can and are actively fixing. So I think I'd rather just not it somewhere, close it, and test it later.

**Comment by mallenexpensify** [comment]
Can't we just use the `[HOLD]` title preface and `Planning` label?
What's he problem we're trying to solve by closing known bugs that aren't fixed?

**Comment by JmillsExpensify** [comment]
I'm not super passionate. Feel free to keep for now. We can circle back when API Unchained is done and bugs/performance and the like are our top priority for WAQ.

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (Second week)

**Comment by AndrewGable** [comment]
Still on hold.

**Comment by JmillsExpensify** [comment]
Added myself so that I can keep track of this one during the sequence numbers implementation.

**Comment by JmillsExpensify** [comment]
Still holding until we get sequence numbers out.

**Comment by JmillsExpensify** [comment]
Still waiting on sequence numbers.

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (Third week)

**Comment by mvtglobally** [comment]
@JmillsExpensify @AndrewGable 
We noticed slightly different behavior in staging today. Could it be related to the same issue reported here? 
https://github.com/Expensify/App/pull/12886#issuecomment-1325307889

**Comment by JmillsExpensify** [comment]
Yes, same issue. Commented in the linked PR.

**Comment by JmillsExpensify** [comment]
Still held on sequence numbers.

**Comment by JmillsExpensify** [comment]
Still on hold for SN, though hopefully we'll be through that and ready and test in the coming week or two.

**Comment by JmillsExpensify** [comment]
Held on sequence numbers.

**Comment by JmillsExpensify** [comment]
Still on hold for SN.

**Comment by JmillsExpensify** [comment]
Unable to reproduce now, and this is actually similar to another recently closed issue https://github.com/Expensify/App/issues/10615. Closing and adding the needs reproduction label.
